app/javascript/application.js

```diff
@@ -2,5 +2,6 @@
 import "@hotwired/turbo-rails"
 import "controllers"
 import Rails from "@rails/ujs"
+import './utils/debounce'
```

Comment:
@robacarp I'm not sure if this is the best pattern for including global utilities or not

Reply:
The "Rails 7 Way" as far as I can tell is this:

- Use ES7 module syntax and import it when you need it and where you need it; the browser will take care of loading it only once when the javascript files all come down from the server.
- Register the utility in the importmap so that Rails renders a module preload script tag in the layout, so you don't have cascade module loading (A loads B loads C loads D).